packages/appsync-emulator-serverless/bin/server.js

```diff
@@ -61,7 +66,7 @@ const main = async () => {
   const dynamodb = dynamoEmulator.getClient(emulator);
 
   const serverless = path.join(path.dirname(pkgPath), 'serverless.yml');
-  const server = await createServer({ serverless, port, dynamodb });
+  const server = await createServer({ wsport, serverless, port, dynamodb });
   // eslint-disable-next-line no-console
   console.log('started at url:', server.url);
   if (dynamodbPort) {
```

Comment:
Please remove `package-lock` file

Reply:
but according to [docs](https://docs.npmjs.com/files/package-lock.json)

> This file is intended to be committed into source repositories, and serves various purposes

Reply:
Do you mind if we use `yarn` instead of `npm` here?

Then we only need `yarn.lock` in the repo root

Reply:
ok, I see